### DISCUSSION TITLE
How to use TOC in custom page?

### DISCUSSION THREAD

**Discussion by Samrose-Ahmed** [facebook/docusaurus#8633] [Q&A]
Is it possible to use the TOC standalone in a .tsx page file and specify the items.

I see this (https://github.com/facebook/docusaurus/issues/2712) was marked as completed but couldn't find anything in the docs.

**Comment by slorber** [comment]
You can try to reuse the `@theme/TOC` component that we use on Markdown content.

The API is the same as `@theme/TOCInline`: https://docusaurus.io/docs/markdown-features/toc#inline-table-of-contents
